spec/io_spec.rb

```diff
@@ -26,7 +26,7 @@
 #
 require "tmpdir" # Used to avoid cluttering the repository.
 require_relative 'spec_helper'
-require_relative "../lib/nmatrix"
+require "./lib/nmatrix"
```

Comment:
I meant 100% tests for NMatrix-JRuby. I agree that JRuby - MRI should support 100% tests. 

I will write travis tests  for JRuby now.